vms/authentication/templates/registration/password_change_form.html

```diff
@@ -0,0 +1,42 @@
+{% extends "vms/base.html" %}
+{% block content %}
+<div id="content-main">
+
+<form action="" method="post">{% csrf_token %}
+<div>
+{% if form.errors %}
+    <p class="errornote">
+    {% if form.errors.items|length == 1 %}Please correct the error below.{% else %}Please correct the errors below.{% endif %}
```

Comment:
You may would want to indent these.

Reply:
Done with indenting and rebasing. Please review.